crates/tabby/src/serve.rs

```diff
@@ -106,8 +106,13 @@ pub struct ServeArgs {
     parallelism: u8,
 
     #[cfg(feature = "ee")]
-    #[clap(hide = true, long, default_value_t = false)]
+    #[clap(hide = true, long, default_value_t = true)]
+    #[deprecated]
```

Comment:
Require that this flag cannot be combined with `no_webserver` and issue a deprecation warning on CLI when using this flag

Reply:
this flag is no longer used (that's why it's marked as deprecated).

The only purpose of keeping it to avoid tabby throws an error when somewhy still pass `--webserver` to it. (forward compatibility)

Reply:
Think we should at least add a CLI deprecation warning, in case some script is still using this automatically it will inform them that it may eventually be removed